### PR TITLE
feat: #30 Allow for variable-width columns and more than two columns

### PR DESCRIPTION
**Pull request by andrew-polk**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/docu-notion/32)

BREAKING CHANGE: Makes changes to the expected css for formatting columns. So clients must make those changes to their respective css files to get the desired results.

### CODE REVIEW

File: src/transformers/ColumnTransformer.ts [hunk 1]
```diff
@@ -21,9 +22,29 @@ export async function notionColumnToMarkdown(
...
+  const style = await getStyleInJsxFormat(id);
+
   // note: it would look better in the markup with \n, but that
   // causes notion-to-md to give us ":::A" instead of \n for some reason.
-  return `<div class='notion-column'>\n\n${childrenStrings.join(
+  return `<div class='notion-column' style=${style}>\n\n${childrenStrings.join(
     "\n\n"
-  )}\n\n</div>`;
+  )}\n\n</div>
+  <div className='notion-spacer' />`;
```
Comment (hatton) [inline src/transformers/ColumnTransformer.ts:32]: Could use a comment

**Review by hatton — APPROVED** [review]

**Review by andrew-polk — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/docu-notion/32)* status: 0 of 4 files reviewed, 3 unresolved discussions (waiting on @hatton)

___
*[`src/transformers/ColumnTransformer.ts` line 32 at r1](https://reviewable.io/reviews/sillsdev/docu-notion/32#-NJkCwyVk_PO66Ss0wn--r1-32:-NJkJLxPD20dmx3xp_8J:b-896fix) ([raw file](https://github.com/sillsdev/docu-notion/blob/da5ee8b4083f4290bbed922fa07476677e5bf16f/src/transformers/ColumnTransformer.ts#L32)):*
<details><summary><i>Previously, hatton (John Hatton) wrote…</i></summary><blockquote>

Could use a comment
</blockquote></details>

Done.
___
*[`src/transformers/ColumnTransformer.ts` line 35 at r1](https://reviewable.io/reviews/sillsdev/docu-notion/32#-NJkCwyVk_PO66Ss0wn--r1-35:-NJkJMDh0mG3NvIUaHxt:b-896fix) ([raw file](https://github.com/sillsdev/docu-notion/blob/da5ee8b4083f4290bbed922fa07476677e5bf16f/src/transformers/ColumnTransformer.ts#L35)):*
<details><summary><i>Previously, hatton (John Hatton) wrote…</i></summary><blockquote>

I don't think this is a hack
</blockquote></details>

Done.
___
*[`src/transformers/ColumnTransformer.ts` line 40 at r1](https://reviewable.io/reviews/sillsdev/docu-notion/32#-NJkCwyVk_PO66Ss0wn--r1-40:-NJkJRItAwzPfvyz1OcH:b-hii3ja) ([raw file](https://github.com/sillsdev/docu-notion/blob/da5ee8b4083f4290bbed922fa07476677e5bf16f/src/transformers/ColumnTransformer.ts#L40)):*
<details><summary><i>Previously, hatton (John Hatton) wrote…</i></summary><blockquote>

This is fine, but consider if it would be cleaner to just have this as `getColumnWidthPixels()` and then have 
```
style=`width=${widthInPixels}px`
```
</blockquote></details>

We want it as a percentage so it flexes with the size of the window like Notion does.
But I took your point about returning the width rather than the whole style.

**Comment by andrew-polk** [comment]
Converting to draft because running this on bloom-docs revealed issues.

**Review by hatton — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/docu-notion/32)* status: 0 of 4 files reviewed, 3 unresolved discussions (waiting on @andrew-polk)

___
*[`src/transformers/ColumnTransformer.ts` line 40 at r1](https://reviewable.io/reviews/sillsdev/docu-notion/32#-NJkCwyVk_PO66Ss0wn--r1-40:-NJkZbaM2f1AmTpAM9gi:b2ptu3w) ([raw file](https://github.com/sillsdev/docu-notion/blob/da5ee8b4083f4290bbed922fa07476677e5bf16f/src/transformers/ColumnTransformer.ts#L40)):*
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

We want it as a percentage so it flexes with the size of the window like Notion does.
But I took your point about returning the width rather than the whole style.
</blockquote></details>

Oh, sorry, and before it was a just a number, but now it is a CSS expression, so the simplification benefit mostly went away :-) Ah well.

**Comment by andrew-polk** [comment]
> Converting to draft because running this on bloom-docs revealed issues.

Ok, the problem is that the css changes have to be copied to the client apps.... which makes this somewhat of a breaking change.
I need to figure out the best way to document that. 
And I'll have to fix paratext-manual, too, I guess.

**Pull request merged**